### DISCUSSION TITLE
Logbook card: add an option to ignore "unavalable" states

### DISCUSSION THREAD

**Discussion by ildar170975** [home-assistant/frontend#16523] [Dashboard]
Some integrations provide entities which may become `unavailable` sometimes.
It may happen due to corresponding external services, network issues etc.

Here is an example with a Starline integration (car security system): sometime entities become `unavailable`:

![изображение](https://github.com/home-assistant/frontend/assets/71872483/5e17a576-4f16-47fa-8531-1da1e19432c8)

Note that in this example a number of occurrences of `unavailable` values is not too much - but it could be much more.
But sometimes these "bad entries" make an analysis more difficult.

One more example - here we have 4 "bad" entries, 6 "good" entries:

![изображение](https://github.com/home-assistant/frontend/assets/71872483/640aa9b3-bcc2-4b49-ab7e-83042fea299d)

**Proposal:**
For a Logbook card add a new option like `ignore_unavailable` (`false` by default) - and do not display `unavailable` entries if `true`.
For a big Logbook page - add a corresponding checkbox in UI.

**Comment by wwwjscom** [comment]
My logbook is also filled with unavailable spam from many different entities.  Completely excluding these entities from my logbook is not ideal.  However, excluding just their unavailable states, and corresponding reporting of being available again states, would be helpful for log analysis.

**Comment by borodaaaaa** [comment]
+1 it is super useful with z2m as it throws in a lot of unavailable entities when reboots flooding the useful log with false positives…

**Comment by borodaaaaa** [comment]
May be an option to ignore some states….
